pkg/webui/lib/field-description-messages.js

```diff
@@ -98,83 +99,250 @@ const m = defineMessages({
   rx1DataRateOffsetDescription:
     'The Data Rate Offset sets the offset between the uplink data rate and the downlink data rate used to communicate with the End Device during the first reception slot (RX1).',
 
-  dataRateIndexDescription:
-    'The Data Rate Index specifies which data rate downlink communications will use, as given in the Regional Parameters.',
+  deviceBrandDescription: 'This is the manufacturer of your end device.',
+  deviceModelDescription: 'The particular model of your end device.',
+
+  deviceHardwareVersionDescription: 'The hardware version of your device.',
+  deviceHardwareVersionLocation:
+    'It should be provided by the manufacturer of your device, or printed on the device packaging.',
+  deviceHardwareVersionAbsence: 'Contact the manufacturer or reseller of your device.',
+
+  deviceFirmwareVersionDescription: 'The version of firmware loaded on your device.',
+  deviceFirmwareVersionLocation:
+    'The firmware version should be provided by the manufacturer of your device, or printed on the device packaging. It may be possible to upgrade your device firmware to a known version.',
+  deviceFirmwareVersionAbsence: 'Contact the manufacturer or reseller of your device.',
+
+  activationModeDescription:
+    'OTAA is the preferred and most secure way to connect a device. Devices perform a join-procedure with the network. ABP requires hardcoding the device address and security keys. Multicast is a virtual group of ABP devices which allows all devices to receive the same downlinks. Multicast groups do not support uplinks.',
+  activationModeLocation: 'You decide how to activate your devices. Whenever possible, use OTAA.',
+
+  deviceNameDescription: 'An optional human readable name to help you identify your device.',
+  deviceNameLocation: 'You make it up, so be creative!',
+
+  deviceDescDescription:
+    'An optional description, which can also be used to save notes about the end device.',
+
+  frameCounterWidthDescription:
+    'Most devices use a 32 bit frame counter to prevent replay attacks. Devices with extremely limited resources are permitted to use 16 bit counters.',
+  frameCounterWidthLocation: 'It should be provided by the device manufacturer.',
+  frameCounterWidthAbsence:
+    'Contact your manufacturer or reseller. Most devices use 32 bit counters. Selecting the wrong value will produce errors once the Up or Down frame counter exceeds 16 bits and rolls over.',
+
+  rx2DataRateIndexDescription: 'The data rate used for the RX2 receive window.',
+  rx2DataRateIndexLocation:
+    'This should be provided by the device manufacturer. It will also automatically configured by the network for OTAA devices.',
```

Comment:
```suggestion
    'This should be provided by the device manufacturer.',
```

I think we can remove the sentence since the field will not be shown for OTAA devices anyway.